src/Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X/RazorExtensions.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Razor.Language;
+using Microsoft.AspNetCore.Razor.Language.Extensions;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X
+{
+    public static class RazorExtensions
+    {
+        public static void Register(IRazorEngineBuilder builder)
+        {
```

Comment:
Explode with a meaningful error if `builder.DesignTime` is `false`. We don't want people to think they can use this to generate 1.0 runtime code.